src/test/java/StringTest.java

```diff
@@ -0,0 +1,46 @@
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+// 
+import static org.assertj.core.api.Assertions.*;
+
+public class StringTest {
+
+  @Test
+  @DisplayName("'1,2' 로 split 시 1, 2로 잘 분리되는지 테스트")
+  public void splitTest1() {
+    String[] str = "1,2".split(",");
+    assertThat(str).contains("1");
+    assertThat(str).containsExactly("1", "2");
+  }
+
+  @Test
+  @DisplayName("'1' 로 split 시 1만 포함하는 배열이 반환되는지 테스트")
+  public void splitTest2() {
+    String[] str = "1".split(",");
+    assertThat(str).contains("1");
+    assertThat(str).doesNotContain("2");
+  }
+
+  @Test
+  @DisplayName("'(1,2)' 값이 주어졌을 때 String 의 substring() 매소드를 활용해 () 을 제거하고 '1,2' 를 반환하도록 테스트")
+  public void substringTest1() {
+    String str = "(1,2)".substring(1, 4);
+    assertThat(str).contains("1, 2");
+  }
```

Comment:
테스트가 깨지는것 같아요.
아래처럼 작성하면 좋겠네요 😄

```suggestion
  @Test
  @DisplayName("'(1,2)' 값이 주어졌을 때 String 의 substring() 매소드를 활용해 () 을 제거하고 '1,2' 를 반환하도록 테스트")
  public void substringTest1() {
    String str = "(1,2)".substring(1, 4);
    assertThat(str).contains("1,2");
  }
```